openverifiablellm/tokenizer/base.py

```
@@ -19,13 +19,30 @@ def __init__(self, vocab_size: int, min_frequency: int):
 
     @abstractmethod
     def train(self, text_file: Path, save_path: Path):
-        """Train tokenizer and save model."""
+        """Train tokenizer on a text corpus and save artifacts to save_path."""
+        pass
+
+    @abstractmethod
+    def encode(self, text: str) -> list:
+        """Encode text into a list of integer token ids."""
+        pass
```

Ruff (0.15.5)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)